src/Clients/Event/Events/AbstractRequestEvent.php

```diff
@@ -10,7 +10,9 @@
 abstract class AbstractRequestEvent
 {
     public readonly string $id;
+
```

Comment:
@h4kuna Do you like this? I think i would prefer no line?

Reply:
As you wish, I have no opinion.

Ok, I will find rule